project/app/src/main/java/com/example/cmput301w21t25/activities_user/MyUserProfileActivity.java

```diff
@@ -138,14 +138,19 @@ public void updateButton(View view) {
         EditText newEmail = findViewById(R.id.updateEmail);
         String name = newName.getText().toString();
         String email = newEmail.getText().toString();
-        UserManager userManager = new UserManager();
 
-        if (!name.equals("") && !email.equals("")) {
-            userManager.FB_isUnique(name, userID, email, this,  "update");
-        }
-        else {
-            //something is null
-            Toast.makeText(MyUserProfileActivity.this, "Don't leave fields empty!", Toast.LENGTH_SHORT).show();
+        if(name.length()>0 && email.length()>0){
```

Comment:
It looks like the FB_isUnique() call was removed, which we need in here somewhere.